ArduPlane/mode_qland.cpp

```diff
@@ -19,6 +19,11 @@ bool ModeQLand::_enter()
 #if AP_FENCE_ENABLED
     plane.fence.auto_disable_fence_for_landing();
 #endif
+
+    // clear any precland timestamps
+    quadplane.poscontrol.last_velocity_match_ms = 0;
```

Comment:
This is reset in the quadplane mode enter. 

https://github.com/ArduPilot/ardupilot/blob/4a310fb2077e55aaf2557b85a502e3dcd782be51/ArduPlane/quadplane.cpp#L4654

Reply:
thanks, removed